skyportal/handlers/api/group.py

```diff
@@ -149,9 +159,12 @@ def get(self, group_id=None):
                     "admin": has_admin_access_for_group(user, group_id),
                 }
                 for user in group.users
-            ]
+            ] if include_group_users else None
             group = group.to_dict()
-            group['users'] = users
+            if users is not None:
+                group['users'] = users
+            else:
+                group.pop("users", None)
```

Comment:
Is this necessary? I don't think relationships are loaded by default when calling to_dict

Reply:
True, removed.